instance/instance.go

```diff
@@ -258,10 +287,12 @@ func validateInstanceUpdate(expectedState string, currentInstance, instance *mod
 	return nil
 }
 
-func (s *Store) defineInstanceLinks(instance *models.Instance, editionDoc *models.Edition) models.InstanceLinks {
+func (s *Store) defineInstanceLinks(instance *models.Instance, editionDoc *models.Edition) *models.InstanceLinks {
 	stringifiedVersion := strconv.Itoa(instance.Version)
 
-	links := models.InstanceLinks{
+	log.Info("got here 9a", log.Data{"editionDoc": editionDoc.Links, "instance": instance})
```

Comment:
Update this log message to provide some context 😄

Reply:
oops